pkg/master/master.go

```diff
@@ -132,7 +132,9 @@ type Config struct {
 	// ExternalHost is the host name to use for external (public internet) facing URLs (e.g. Swagger)
 	ExternalHost string
 
-	// If nil, the first result from net.InterfaceAddrs will be used.
+	// PublicAddress is the IP address where members of the cluster (kubelet,
+	// kube-proxy, services, etc.) can reach the master.
+	// If nil or 0.0.0.0, the first result from net.InterfaceAddrs will be used.
```

Comment:
I went ahead and just killed the TODO from below. Now, PublicAddress must be set.